volplugin/volplugin.go

```diff
@@ -166,50 +166,56 @@ func (dc *DaemonConfig) getGlobal() {
 
 func (dc *DaemonConfig) watchGlobal() error {
 	for {
-		time.Sleep(1 * time.Minute)
+		time.Sleep(1 * time.Second)
 		dc.getGlobal()
 	}
 }
 
 func (dc *DaemonConfig) updateMounts() error {
-	cd := ceph.NewDriver()
-	mounts, err := cd.Mounted(dc.Global.Timeout)
-	if err != nil {
-		return err
-	}
+	for driverName := range backend.Drivers {
+		cd, err := backend.NewDriver(driverName, dc.Global.MountPath)
```

Comment:
ah, ok..got it.

I saw his code https://github.com/contiv/volplugin/pull/171/files#diff-1094dbd71a8251a1917ea1cbdff6520cR176 which made me think every driver might want to do this or do it differently which made me wonder if it was ok to have a global mountpath. But looking further down that code I see that the volumepath are namespaced, so as long as each driver just restricts to their owned volumes they should be fine.